src/ert/config/gen_kw_config.py

```diff
@@ -223,8 +225,12 @@ def read_from_runpath(
 
     def write_to_runpath(
         self, run_path: Path, real_nr: int, ensemble: EnsembleReader
-    ) -> Dict[str, Dict[str, float]]:
-        array = ensemble.load_parameters(self.name, real_nr)["transformed_values"]
+    ) -> Optional[Dict[str, Dict[str, float]]]:
+        try:
+            array = ensemble.load_parameters(self.name, real_nr)["transformed_values"]
```

Comment:
We should not try to load parameters if a realization has failed, think this might be related to the state map, as that is how we used to handle this.

Reply:
Looked a bit more into it, and with this we try to run all the realizations, even though their parent failed, can be fixed in the run models with:

```                initial_mask=np.logical_and(
                    prior_context.sim_fs.get_realization_mask_with_parameters(),
                    prior_context.sim_fs.get_realization_mask_with_responses(),
                ),
```
currently it is logical_or, but we need both for the posterior

Reply:
This we should remove this `try/except`, as it might mask some other problems.